hyperstart.go

```diff
@@ -222,9 +223,14 @@ func (h *hyper) buildNetworkInterfacesAndRoutes(pod Pod) ([]HyperNetIface, []hyp
 		ifaces = append(ifaces, iface)
 
 		for _, r := range endpoint.Properties.Routes {
+			gateway := r.GW.String()
+			if gateway == "<nil>" {
```

Comment:
Yeah... I don't like that either. Take a look at https://golang.org/pkg/net/#IP.String